dandi/metadata.py

```diff
@@ -408,13 +410,16 @@ def extract_session(metadata: dict) -> list:
     for val in probe_ids:
         probes.append(models.Equipment(identifier=f"probe:{val}", name="Ecephys Probe"))
     probes = probes or None
-    if (metadata.get("session_id") or probes) is None:
+    if (
+        metadata.get("session_id") or metadata.get("session_start_time") or probes
+    ) is None:
```

Comment:
Afaik session id 0 is as legit as any other. I think such shortcut would get us into trouble eventually .

Reply:
this shouldn't evaluate against `None` it's a very specific check to see if the boolean combination is None.

Reply:
there is no explicit test for `None` for anything here but `probes` here pretty much. So if session_id is set and == 0, we would get `None` if probes is None.
```
$> python3 -c 'ses = 0; probes = None; print((ses or probes) is None)'
True
```